### ISSUE TITLE
Uboot env variable usb_ignorelist=0x1050:*,

### ISSUE THREAD

**Issue by vovkapoc** [AsahiLinux/m1n1#388]
Hello! In my uboot env exists variable usb_ignorelist with value '0x1050:*,'

What is this variable? And why does it matter so much?

0x1050 is YubiKey vendor ID, but after this * is all vendor IDs??

**Comment by jannau** [comment]
no, the `*` the is the wildcard for all product IDs from Yubico. Yubikeys are ignored since they present themself as USB keyboard. U-boot supports just a single one so it should not be a Yubikey.

See https://lore.kernel.org/u-boot/20240404-asahi-keyboards-v4-4-2266033feaff@jannau.net/ for details.

This is the wrong place to ask for this, m1n1 is mostly unrelated to u-boot.
